src/js/components/Drop/stories/Overflow.js

```diff
@@ -19,44 +18,45 @@ const OverflowDrop = () => {
   useEffect(() => setShowDrop(true), []);
 
   return (
-    <Grommet theme={grommet} full>
```

Comment:
@jcfilben just pointing out another³ case of `full`, as you mentioned it on another PR.

Reply:
Same situation, I think we could either have the `full` prop or add padding to the box container so the contents are more centered.